sdk/core/Azure.Core.TestFramework/src/RecordedTestAttribute.cs

```diff
@@ -9,11 +9,11 @@
 
 namespace Azure.Core.TestFramework
 {
-    [AttributeUsage(AttributeTargets.Method)]
     /// <summary>
     /// This attribute replaces the [Test] attribute and will dynamically re-record recorded tests on failure.
     /// Tests that are re-recorded will complete with a error status and indicate that copying the updated recording to SessionRecords is needed.
     /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
```

Comment:
What if a test doesn't use the `RecordedTestAttribute ` ?

Reply:
Then it will get the global timeout.

Reply:
Even for a live run? Am I correct that this makes the [RecordedTest] attribute mandatory?